chapters/06-smart-contracts-in-ralph.md

```diff
@@ -2592,7 +2599,7 @@ Compared to the contract events, which are defined and emitted explicitly from t
 
 ```rust
 TxScript Deploy(fooByteCode: ByteVec) {
-  createContract!{callerAddress!() -> ALPH: 1 ALPH}(fooByteCode, #00, #00)
+  let _ = createContract!{callerAddress!() -> ALPH: minimalContractDeposit!()}(fooByteCode, #00, #00)
```

Comment:
@h0ngcha0 I was getting a compiler error and this is the solution it recommended